pmm-app/src/pmm-add-instance-app-panel/AddInstance/AddRemoteInstance/AddRemoteInstanceService.tsx

```diff
@@ -31,8 +31,6 @@ class AddRemoteInstanceService {
         return AddRemoteInstanceService.addProxysql(data);
       case 'PostgreSQL':
         return AddRemoteInstanceService.addPostgresql(data);
-      default:
```

Comment:
I expected throwing some error or showing some message instead of just ignoring it

Reply:
Added exception for now, in future it will be replaced with TS checks